src/app/vendors/SingleVendor.tsx

```diff
@@ -177,11 +178,12 @@ function SingleVendor({
     if (artifactDisplay) {
       displayName = artifactDisplay.name;
       displayDesc = artifactDisplay.description;
+      artifactCheck = 'seasonalArtifact';
     }
   }
 
   return (
-    <div className={clsx(styles.page, 'dim-page')}>
+    <div className={clsx(styles.page, `dim-page ${artifactCheck}`)}>
```

Comment:
```suggestion
    <div className={clsx(styles.page, 'dim-page', artifactCheck)}>
```

Reply:
ohhhhhh thats how :P

Reply:
now that you're using clsx properly you can go back to
`let artifactCheck:string|undefined;` :)

Reply:
great point, resolved as well!